errtrace.go

```diff
@@ -118,3 +122,20 @@ func (e *errTrace) Format(s fmt.State, verb rune) {
 
 	fmt.Fprintf(s, fmt.FormatString(s, verb), e.err)
 }
+
+// TracePC returns the program counter for the location in this frame.
+func (e *errTrace) TracePC() uintptr {
+	return e.pc
+}
+
+// tracePCprovider is a provider of the Program Counter
+// that the error originated with.
+// The returned PC is intended to be used with
+// runtime.CallersFrames or runtime.FuncForPC
+// to aid in generating the error return trace
+type tracePCprovider interface {
```

Comment:
do we want this interface -- we're not using it anywhere other than to that `errTrace` has the right method. We can probably drop it, or if we keep it, we should use it in `UnwrapFrame`

tangentially related nit, with camelcase I'd expect this to be `tracePCProvider` (though that is a little confusing to read). if we decide to keep it, we may want to rename it

Reply:
No worries. I will get rid of it now that it is unexported.